src/core/DoomCounter.ttslua

```diff
@@ -0,0 +1,75 @@
+local optionsVisible = false
+local options = {
+  Agenda = true,
+  Playarea = true,
+  Playermats = true
+}
+
+val = 0
+
+-- save current value and options
+function onSave() return JSON.encode({ val, options }) end
+
+function onLoad(savedData)
+  if savedData ~= "" then
+    local loadedData = JSON.decode(savedData)
+    val              = loadedData[1]
+    options          = loadedData[2]
+
+    -- restore state for option panel
+    for key, bool in pairs(options) do
+      if bool then
+        self.UI.setAttribute("option" .. key, "isOn", 0)
```

Comment:
Save/load doesn't seem to be working for me.  Haven't tested exhaustively, but after changing a setting, overwriting, and then reloading it resets to all options being chosen.

I think isOn may need a direct true/false here, rather than the converted 0-is-false?

I think you can just set the value of bool to each for all the items as well to simplify a bit